src/app/form/form.component.html

```diff
@@ -0,0 +1,81 @@
+                    </span>
+                        <span class="last-name">
+                      <input type="text" class="form-control" name="last-name" formControlName="studentLastName">
+                      <label class="form-sub-info-text">Last</label>
+                    </span>
+                    </div>
+                    <div class="clearfix"></div>
+                    <div class="warning" *ngIf="currentStudent.controls['studentFirstName'].invalid && currentStudent.controls['studentFirstName'].touched || 
+                    currentStudent.controls['studentLastName'].invalid && currentStudent.controls['studentLastName'].touched">
+                        Enter a value for this field.
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                    <span>Phone Number</span>
+                    <em class="required-star">*</em>
+                  </label>
+                    <input type="tel" class="form-control" name="phone" formControlName="studentPhone">
+                    <div class="warning" *ngIf="currentStudent.controls['studentPhone'].invalid && currentStudent.controls['studentPhone'].touched">
+                        Enter a number for this field like this (+380) 63-889-11-89.
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                  <span>Email Address</span>
+                </label>
+                    <input type="email" class="form-control" name="email" formControlName="studentEmail">
+                    <div class="warning" *ngIf="currentStudent.controls['studentEmail'].invalid && currentStudent.controls['studentEmail'].touched">
+                        Enter a valid email address. (eg: yourname@domain.com)
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+                <span>University / College of attendance</span>
+              </label>
+                    <input type="text">
+                </div>
+
+                <div class="form-group">
+                    <label class="label">
+              <span>Your Message</span>
+            </label>
+                    <textarea></textarea>
+                </div>
+
+                <div class="btn-submit">
+                    <input type="submit" value="Submit" [disabled]="currentStudent.invalid" [ngClass]="{'invalidForm': isInvalidForm}" *ngIf="currentStudent.invalid; else unset">
+                    <ng-template #unset>
+                        <input type="submit" value="Submit" [ngClass]="{'validForm': isValidForm}">
+                    </ng-template>
+                </div>
+            </form>
+            <p class="spacesBelowForm">-----------------------------------------------------------------------------------------------------------------------------</p>
```

Comment:
what is this?

Reply:
![image](https://user-images.githubusercontent.com/62300360/100355536-01503180-3003-11eb-9c4e-e7cf781292bc.png)

Reply:
It's a line under button

Reply:
ok